docs/parse_reference.rst

```diff
@@ -50,4 +50,21 @@ Parser
 
       :raise: :exc:`ValueError` if *image* has invalid format.
 
-.. _DockerHub: https://hub.docker.com
+  .. _DockerHub: https://hub.docker.com
+
+   .. method:: env(env: Sequence[str], env_file: Sequence[str] = ()) -> Tuple[Dict[str, str], Mapping[str, URL]]
+
+      Parse a sequence of *env* variables and a sequence of *env_file* file names into a
+      tuple of two mappings - first one for all environment variables and second for
+      environment variables using secrets.
+
+      :raise: :exc:`ValueError` if caller attempts to redefine a system-reserved env var
+
+   .. method:: volumes(volume: Sequence[str]) -> Tuple[Set[Volume], Set[SecretFile]]
+
+      Parse a sequence of volume definition into a tuple of two mappings - first one for
+      all regular volumes and second one for volumes using secrets.
+
+      :raise: :exc:`ValueError` if `volumes` sequence contains `ALL` or `HOME` values, which are deprecated.
```

Comment:
I think it is not needed. First, they are not deprecated, they are errors. Second, a ValueError is raised for any other invalid volume specification.